x/oracle/abci_test.go

```diff
@@ -227,11 +227,12 @@ func (s *IntegrationTestSuite) TestEndblockerHistoracle() {
 			// query for past 6 medians (should only get 4 back since max median stamps is set to 4)
 			medians := app.OracleKeeper.HistoricMedians(ctx, denom.SymbolDenom, 6)
 			s.Require().Equal(4, len(medians))
-			s.Require().Equal(tc.expectedHistoricMedians, medians)
+
+			s.Require().Equal(tc.expectedHistoricMedians, medians.Decs())
 
 			medianHistoricDeviation, err := app.OracleKeeper.HistoricMedianDeviation(ctx, denom.SymbolDenom)
 			s.Require().NoError(err)
-			s.Require().Equal(tc.expectedHistoricMedianDeviation, medianHistoricDeviation)
+			s.Require().Equal(tc.expectedHistoricMedianDeviation, medianHistoricDeviation.ExchangeRateTuple.ExchangeRate)
```

Comment:
We might want to check that the block num set in these medians is as expected in this test (not just the price), but that could be in another PR down the road.

Reply:
I agree, we could do that when this file gets cleaned up and some of this logic is moved elsewhere.